### PR TITLE
Serverless functions

### PR DESCRIPTION
**Pull request by briancbarrow**

### CODE REVIEW

**Review by lukeocodes — CHANGES_REQUESTED** [review]
As discussed, please use Zapier for the integration with hellonext. This is also missing the components from shared, including the forms.

**Comment by briancbarrow** [comment]
> This is also missing the components from shared, including the forms.

What do you mean by this? This PR is just for the serverless functions. I'll add the UI code in a separate PR.

File: serverless/netlify/functions/general-zap/index.js [hunk 1]
```diff
@@ -0,0 +1,17 @@
+const axios = require('axios')
```
[the remark this answers is not in the document]
Comment (briancbarrow) [reply serverless/netlify/functions/general-zap/index.js:1]: yep

File: serverless/netlify/functions/forms/index.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    data: { data: streams },
+  } = await axios.post(
+    url,
+    event.body
+    
+  )
+    return {
+      statusCode: 200,
+    }  
+}
```
Comment (lukeocodes) [inline serverless/netlify/functions/forms/index.js:17]:
```suggestion
const axios = require("axios");

exports.handler = async function (event, context) {
	const body = JSON.parse(event.body);
	const hookId = body.hookId;
	const url = `${process.env.ZAP_HOOK_URL}${hookId}`;
	const {
		data: { data: streams },
	} = await axios.post(url, event.body);
	return {
		statusCode: 200,
	};
};

```

**Review by lukeocodes — CHANGES_REQUESTED** [review]
One last change. I linted it for you. Can you configure prettier on your IDE so it runs on save? Config is all set up already

File: serverless/netlify/functions/forms/index.js [hunk 3]
```diff
@@ -1,13 +1,16 @@
...
+  const body = JSON.parse(event.body);
+  const hookId = body.hookId;
+  const url = `${process.env.ZAP_HOOK_URL}${hookId}`;
+  const {
+    data: { data: streams },
+  } = await axios.post(url, event.body);
+  return {
+    statusCode: 200,
+    headers: {
+      "access-control-allow-origin": "*",
```
Comment (lukeocodes) [inline serverless/netlify/functions/forms/index.js:13]: We may need to update this to our domains when the sites are launched

**Review by MichaelJolley — APPROVED** [review]

**Pull request merged**